docs/authoring-components.md

```diff
@@ -316,8 +310,16 @@ Since this component is a vanilla component, it should be modeled after the vani
 favors getters and setters to implement its functionality (think `checked`, `disabled`, etc.). Our
 adapter is extremely straightforward as we can simply repurpose the methods we started out with.
 
-```js
+```ts
 class RedblueToggle extends MDCComponent {
+  initialize() {
+    this.root.addEventListener('click', this.foundation_.handleClick);
```

Comment:
These can use `this.listen` / `this.unlisten` here now that the class extends MDCComponent